src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneCommands/ExportAzureRmDataLakeStoreItem.cs

```diff
@@ -130,63 +128,55 @@ public LogLevel DiagnosticLogLevel
 
         public override void ExecuteCmdlet()
         {
+            if (ConcurrentFileCount != -1)
+            {
+                WriteWarning(Resources.IncorrectConcurrentFileCountWarning);
```

Comment:
@rahuldutta90 rather than writing this warning, put these strings inside their corresponding `Obsolete` attribute

Reply:
Fixed.

Reply:
@cormacpayne Do you have any other comments regarding this?

Reply:
@cormacpayne , I have maked obsolete. However, that alone does not show the message when running the cmdlet. So I am leaving the WriteWarning in as well.